opds.py

```diff
@@ -110,12 +110,31 @@ def __init__(self, base_url, base_filename, default_order=None):
             base_filename = base_filename[:-5]
         self.base_filename = base_filename
 
-    def facet_url(self, facets):
+    def default_lane_url(self):
+        return self.base_url + '/' + self.base_filename + '.opds'
+
+    def filename_facet_segment(self, facets):
         ordered_by = list(facets.items())[0][1]
+        if ordered_by != self.default_order:
+            return '_' + ordered_by
+        return ''
 
+    def facet_url(self, facets):
+        """Incoporate order facets into filenames for static feeds
+        """
         filename = self.base_filename
-        if ordered_by != self.default_order:
-            filename += ('_'+ordered_by)
-        filename += '.opds'
+        filename += self.filename_facet_segment(facets)
+
+        return self.base_url + '/' + filename + '.opds'
+
+    def feed_url(self, lane, facets, pagination):
+        """Incorporate pages into filenames for static feeds
+        """
+        filename = self.base_filename
+        filename += self.filename_facet_segment(facets)
+
+        page_number = (pagination.offset / pagination.size) + 1
+        if page_number > 1:
+            filename += ('_%i' % page_number)
```

Comment:
Yes, it's used in `AcquisitionFeed.page` when the "next" and "previous" links are added to the feed.